src/components/ModalContent/ModalContent.module.scss

```diff
@@ -0,0 +1,10 @@
+@import "../../styles/themes";
+@import "../../styles/states";
+
+.mondayStyleModalContent {
```

Comment:
we usually do not add monday prefix when using css modules as standard

Reply:
👍